op-devstack/presets/minimal_external_el.go

```diff
@@ -0,0 +1,65 @@
+package presets
+
+import (
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/shim"
+	"github.com/ethereum-optimism/optimism/op-devstack/stack"
+	"github.com/ethereum-optimism/optimism/op-devstack/stack/match"
+	"github.com/ethereum-optimism/optimism/op-devstack/sysgo"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+type MinimalExternalEL struct {
+	Log          log.Logger
+	T            devtest.T
+	ControlPlane stack.ControlPlane
+
+	L1Network *dsl.L1Network
+	L1EL      *dsl.L1ELNode
+
+	L2Chain *dsl.L2Network
+	L2CL    *dsl.L2CLNode
+
+	SyncTester *dsl.SyncTester
+
+	Wallet *dsl.HDWallet
+}
+
+func (m *MinimalExternalEL) L2Networks() []*dsl.L2Network {
+	return []*dsl.L2Network{
+		m.L2Chain,
+	}
+}
+
+func (m *MinimalExternalEL) StandardBridge() *dsl.StandardBridge {
+	return dsl.NewStandardBridge(m.T, m.L2Chain, nil, m.L1EL)
+}
+
+func WithMinimalExternalELWithSuperchainRegistry(l1CLBeaconRPC, l1ELRPC, l2ELRPC string, l1ChainID eth.ChainID, networkName string, fcus eth.FCUState) stack.CommonOption {
```

Comment:
Very nit and nonblocking:
```suggestion
func WithMinimalExternalELWithSuperchainRegistry(l1CLBeaconRPC, l1ELRPC, l2ELRPC string, l1ChainID eth.ChainID, networkName string, fcu eth.FCUState) stack.CommonOption {
```